index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <body class="bk-Color">
```

Comment:
- [ ] Please include an explanation for your requirement, in this case, you can include something like: "An HTML document can only have one body tag. It could cause your document to be invalid, and the browsers may have problems displaying it".

Reply:
- [ ] Please include an explanation for your requirement, in this case, you can include something like: "An HTML document can only have one body tag. It could cause your document to be invalid, and the browsers may have problems displaying it".